src/main/resources/templates/aeolus/c_sharp/default.sh

```
@@ -0,0 +1,26 @@
+#!/usr/bin/env bash
+set -e
+export AEOLUS_INITIAL_DIRECTORY=${PWD}
+build () {
+  echo '⚙️ executing build'
+  dotnet build "${testWorkingDirectory}"
```

Shellcheck

[warning] testWorkingDirectory is referenced but not assigned.

(SC2154)